### PR TITLE
Add section to the developer page and link to it in the r frnr0f8

### PR DESCRIPTION
**Pull request by M-Smeby**
- [ ] [API update message](https://github.com/bring/developer-site#tips-for-writing-an-api-update-message) (changelog entry) has been reviewed by Mybring Experience.

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/rFRNR0f8](https://trello.com/c/rFRNR0f8)

File: content/api/tracking/index.md [hunk 1]
```diff
@@ -11,6 +11,11 @@ menu:
 weight: 31
  
 important:
+  - type: warn
+    title: Rate limiting our unauthenticated users - 20.03.2023
```
Comment (stianste) [inline content/api/tracking/index.md:15]:
```suggestion
    title: Rate limiting unauthenticated users - 20.03.2023
```

File: content/api/tracking/index.md [hunk 2]
```diff
@@ -11,6 +11,11 @@ menu:
...
+    message: |
+      Starting 20 March 2023, the Tracking API will start rate limiting our unauthenticated users. This will happen gradually for the next two months.
+      Ending with Tracking API only being available for authenticated users, to read more about this look at the authentication section
```
Comment (stianste) [inline content/api/tracking/index.md:18]:
```suggestion
      After these two months have passed, the Tracking API will only be available for authenticated users. To read more about how to authenticate, see the authentication section
```

Comment (stianste) [reply content/api/tracking/index.md:18]: Also, is it possible to link to the authentication section?

File: content/api/tracking/index.md [hunk 3]
```diff
@@ -31,6 +36,9 @@ information:
 
       To make authenticated Tracking API requests: You will need an API key from Mybring. Steps for getting a key and description of headers can be found on the general API [Getting Started / Authentication](/api/#authentication) page.
 
+      The Tracking API will stop being open to anyone and start requiring user to be authenticated. We have started rate limiting our unauthenticated, this will result in
```
Comment (stianste) [inline content/api/tracking/index.md:39]:
```suggestion
      The Tracking API will stop being open to anyone and start requiring user to be authenticated. We have already started rate limiting unauthenticated users, which will result in
```

**Review by stianste — COMMENTED** [review]
Left some suggestions 👍

**Comment by arewg** [comment]
Should it say PickupPoint API in the message? Should be Tracking API right?

**Comment by M-Smeby** [comment]
> Should it say PickupPoint API in the message? Should be Tracking API right?

Yes, a copy paste error

File: content/api/tracking/index.md [hunk 4]
```diff
@@ -11,15 +11,22 @@ menu:
...
-  - type: warn
-    title: Removed event statuses from open tracking - 22.06.2022
-    message: |
-     The event statuses ARRIVED_COLLECTION and ARRIVED_DELIVERY are not valid anymore, and therefore have been removed from the Tracking API.
+- type: warn
+  title: Rate limiting unauthenticated users - 20.03.2023
+  message: |
+     Our customers are increasingly demanding new functionality and solutions that are easy to use.
+     To meet these expectations, we must know which customers and partners are using our APIs and how they are using them.
+     To further develop the value proposition of our Tracking API, the solution becomes authenticated from 01.05.2023.
```
Comment (stianste) [inline content/api/tracking/index.md:19]:
```suggestion
     To further develop the value proposition of our Tracking API, the solution will require authentication from 01.05.2023.
```

**Review by stianste — APPROVED** [review]

**Comment by richas2** [comment]
Just FYI. Irrespective of the content, we cannot merge this PR yet.

**Comment by M-Smeby** [comment]
> Just FYI. Irrespective of the content, we cannot merge this PR yet.

I know, that is why it is kept as a draft

File: content/api/tracking/index.md [hunk 5]
```diff
@@ -11,15 +11,22 @@ menu:
...
-    title: Removed event statuses from open tracking - 22.06.2022
-    message: |
-     The event statuses ARRIVED_COLLECTION and ARRIVED_DELIVERY are not valid anymore, and therefore have been removed from the Tracking API.
+- type: warn
+  title: Rate limiting unauthenticated users - 20.03.2023
+  message: |
+     Our customers are increasingly demanding new functionality and solutions that are easy to use.
+     To meet these expectations, we must know which customers and partners are using our APIs and how they are using them.
+     To further develop the value proposition of our Tracking API, the solution the solution will require authentication from 01.05.2023.
+     Starting 20th of March 2023, the Tracking API will start rate limiting unauthenticated users.
```
Comment (halvorsanden) [inline content/api/tracking/index.md:20]: One suggestion:
- It’s generally a good idea to start off with what’s happening, not the reason, suggesting rewriting the last two paragraphs and moving them to the start: `The Tracking API will start rate limiting unauthenticated users on 20 March 2023 and require authentication from May 1 2023. Our customers […]`
- Notce the date format is the same as the existing message below. Further ref.: https://www.mybring.com/design-system/guidelines/numbers-units/ The date format in the title is fine.
- This also fixes `the solution the solution` which was repeated.

File: content/api/tracking/index.md [hunk 6]
```diff
@@ -11,15 +11,22 @@ menu:
...
-     Starting 1 August 2022, Tracking API version v1 will be deprecated and the requests will be automatically forwarded to the latest version, v2.
-     It is not a breaking change. In the new version you will have access to many more fields.
-  - type: warn
-    title: Removed event statuses from open tracking - 22.06.2022
-    message: |
-     The event statuses ARRIVED_COLLECTION and ARRIVED_DELIVERY are not valid anymore, and therefore have been removed from the Tracking API.
+- type: warn
+  title: Rate limiting unauthenticated users - 20.03.2023
+  message: | 
+   The Tracking API will start rate limiting unauthenticated users on 20 March 2023 and require authentication from May 1, 2023.
```
Comment (stianste) [inline content/api/tracking/index.md:17]: May 1st? That's already been, no?

Comment (M-Smeby) [reply content/api/tracking/index.md:17]: Yes, the date is wrong. Just need a new date, forgot about that

File: content/api/tracking/index.md [hunk 7]
```diff
@@ -11,15 +11,22 @@ menu:
...
-  - type: warn
-    title: Removed event statuses from open tracking - 22.06.2022
-    message: |
-     The event statuses ARRIVED_COLLECTION and ARRIVED_DELIVERY are not valid anymore, and therefore have been removed from the Tracking API.
+- type: warn
+  title: Rate limiting unauthenticated users - 20.03.2023
+  message: | 
+   The Tracking API will start rate limiting unauthenticated users on 20 March 2023 and require authentication from May 1, 2023. 
+   Our customers are increasingly demanding new functionality and solutions that are easy to use. 
+   To meet these expectations, we must know which customers and partners are using our APIs and how they are using them.
```
Comment (arewg) [inline content/api/tracking/index.md:19]: Should we provide some text saying that personal data etc will not be used, or describe how we are using the data? People might have questions about what _"we must know which customers and partners are using our APIs, and how they are using them"_ means, as we are saying we will identify you and look at your usage to identify how we can improve our services.

Comment (M-Smeby) [reply content/api/tracking/index.md:19]: Might be a good idea

Comment (M-Smeby) [reply content/api/tracking/index.md:19]: @richas2 What do you think?

**Pull request merged**